docs/whats_new.rst

```diff
@@ -238,6 +238,50 @@ Miscellaneous
 - Add low-level component example (:issue:`452`)
 - Update Discord server invite links (:issue:`476`)
 
+.. _vp2p4p1:
+
+v2.4.1
+------
+
+This release is a bugfix release with backports from v2.6.0.
```

Comment:
v2.6.0 isn't released yet and will most likely receive more fixes that won't be part of v2.4.1, this should be rephrased

Reply:
Any suggestions? This will be on the latest docs until 2.6 is released, so I don't see an issue with the wording, plus it doesn't say its "all" backports so I don't see an issue there either.

Reply:
Not sure. In the end v2.6.0 will contain fixes that v2.4.1 does not have, so "backports from v2.6.0" would be somewhat misleading.
Perhaps something like `backports of most fixes from newer versions up to v2.6.0`?

Reply:
Ah, I didn't read this wording as including all backports. Nonetheless, this one is technically wrong, as it includes backports of fixes in 2.5.2, not 2.6.0.

Reply:
2.4.1 also contains fixes from 2.5.0, not just 2.5.2, so something like "up to" would be better?

Reply:
```
This release is a bugfix release with backports from v2.5.0 up to v2.5.2.
```